docs/src/main/sphinx/connector/memory.md

```diff
@@ -73,7 +73,7 @@ statements, the connector supports the following features:
 
 ### TRUNCATE and DROP TABLE
 
-Upon execution of a `TRUNCATE` and a `DROP TABLE` operation, memory is not released
+Upon execution of a `DROP TABLE` operation, memory is not released
```

Comment:
> now memory is released immediately (#25564)

I don't think #25564 releases memory immediately. `MemoryPagesStore#purge` method is called from `MemoryPageSinkProvider#createPageSink`. TRUNCATE statement doesn't reach there.

Reply:
Ah, it is actually called next time. Thanks, I got it.

Reply:
I confirmed it. Thank you~ 🙇‍♂️